### ISSUE TITLE
Is it possible to dynamically add or remove controllers?

### ISSUE THREAD

**Issue by kunass2** [kitasuke/PagingMenuController#370]
Can I do it with your library? I really appreciate it. Nice, clean and simple, but all I need yet is to remove and adding controllers dynamically. Number of controllers are strongly related to entities from `NSFetchedResultsController`. There are methods to add, move, delete, update controllers. Is it possible here? Can I ask for such feature?

**Comment by rdgier** [comment]
Yes, I have done this for the app I am working on. Long story short, I call setup again on PagingMenuController with an adjusted paging options.

I am not using an up to date version, but back then this was the only method I could find to get this to work. Hope it helps.
